server/src/main/kotlin/tech/figure/objectstore/gateway/configuration/DataConfig.kt

```diff
@@ -74,12 +74,10 @@ class DataConfig {
 
     @Bean
     fun flyway(dataSource: DataSource, databaseProperties: DatabaseProperties): Flyway = Flyway(
-        FluentConfiguration().dataSource(dataSource).baselineOnMigrate(databaseProperties.baselineOnMigrate).apply {
-            if (databaseProperties.type != "postgresql") {
```

Comment:
The new structure allows the `1_0__Init_UUID` file to be postgres-specific by living in the postgresql directory